### PR TITLE
#7560: Re-enable Stable Diffusion tests in CI

### PR DESCRIPTION
**Pull request by esmalTT**
### Ticket
#7560 

### Summary

It seems like we can re-enable the SD tests with `SLOW_MATMULS` enabled, as I have not seen any hangs or failures after 3 pipeline runs in this configuration. 

Once merged, I will monitor the pipeline for failures.

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: tests/scripts/single_card/nightly/run_wh_b0_unstable.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ fail=0
 
 echo "Running unstable nightly tests for WH B0 only"
 
-SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml env pytest -n auto tests/ttnn/integration_tests/stable_diffusion ; fail+=$?
+env SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto tests/ttnn/integration_tests/stable_diffusion ; fail+=$?
```
Comment (tt-rkim) [inline tests/scripts/single_card/nightly/run_wh_b0_unstable.sh:13]: DAMN!!!

Comment (esmalTT) [reply tests/scripts/single_card/nightly/run_wh_b0_unstable.sh:13]: F

**Comment by esmalTT** [comment]
> how's a run of this looking like for nightly FD?

@tt-rkim These two seem good:

- https://github.com/tenstorrent/tt-metal/actions/runs/10290966554/job/28482916473
- https://github.com/tenstorrent/tt-metal/actions/runs/10290966554/job/28510486420

**Review by tt-rkim — APPROVED** [review]

**Comment by tt-rkim** [comment]
RUN ITTTTT

Do we need to update SD codeowners?

**Comment by esmalTT** [comment]
> RUN ITTTTT
> 
> Do we need to update SD codeowners?

@tt-rkim Good call. I updated it here in a separate commit. Added @mywoodstock as an owner as well since I think he's the most familiar with SD at the moment - let me know if you think someone else is better suited.

File: tests/scripts/single_card/nightly/run_wh_b0_unstable.sh (hunk 1 above)
Comment (ttmchiou) [reply tests/scripts/single_card/nightly/run_wh_b0_unstable.sh:13]: ...

**Pull request merged**